### ISSUE TITLE
A11y: Cannot leave text field of query editor via keyboard

### ISSUE THREAD

**Issue by L-M-K-B** [grafana/grafana#53829]
**Steps to reproduce**:

When the manual query editor is used in a query row in Explore try to leave the text field by using the tab key.

**Actual Result**:

Can’t exit text field 

**Expected Result**

Repeatable UX for those not using a pointer device

**Relevant WCAG Criteria:**

[2.1.2 No Keyboard Trap - Level A](https://www.w3.org/WAI/WCAG21/quickref/#no-keyboard-trap)

**Environment**:
- Grafana version: 9.2.0.
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by Toledodev** [comment]
Hey @L-M-K-B,  can you show screenshots or a loom video reproducing the error? Thanks  :+1:

**Comment by L-M-K-B** [comment]
After consulting with @grafana/observability-experience-squad I close this issue as the text field can be exited by using the esc key.
